src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -152,7 +237,9 @@ const ChatItemMyTextV2 = ({
                 />
 
                 <View style={{flexDirection: 'row'}}>
-                  <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  {attachments.length <= 0 && (
+                    <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The message is now only rendered if there are no attachments. This makes sense as you wouldn't want to display the message text if there's an attachment that should be displayed instead.


<!-- This is an auto-generated comment by OSS CodeRabbit -->